src/matter/interaction/InteractionMessenger.ts

```diff
@@ -190,7 +190,20 @@ export class InteractionServerMessenger extends InteractionMessenger<MatterDevic
             }
         }
 
-        await this.exchange.send(MessageType.ReportData, TlvDataReport.encode(dataReport));
+        try {
```

Comment:
if (dataReport.suppressResponse) {
  tryCatch(() => await this.exchange.send(MessageType.ReportData, true),
     UnexpectedMessageError, error => this.throwIfError(messageType, 
message.payload);
    // add comment
} else {
   await this.exchange.send(MessageType.ReportData);
   await this.waitForSuccess();
}